### ISSUE TITLE
Invalid parameter list with onDuplicateKeyUpdate with empty values

### ISSUE THREAD

**Issue by ErwinSteffens** [envms/fluentpdo#230]
We have an issue with the `onDuplicateKeyUpdate` when using empty/zero/false values in the update list. The following query re-produces the problem:

```
$inserts = [
    'id'      => "1",
    'column1' => "value1",
    'column2' => false,
];

$updates = [
    'column1' => "value1",
    'column2' => false,
];

$query = $queryHelper->fluent()
                     ->insertInto( 'my_table', $inserts )
                     ->onDuplicateKeyUpdate( $updates );

var_dump($query->getQuery(true));
var_dump($query->getParameters());

$query->execute();
```

The generated query is:
```
INSERT INTO my_table (id, column1, column2)
VALUES (?, ?, ?)
ON DUPLICATE KEY UPDATE column1 = ?, column2 = ?' (length=109)
```

The parameter list results in this:
```
array (size=4)
  0 => string '1' (length=1)
  1 => string 'value1' (length=6)
  2 => boolean false
  3 => string 'value1' (length=6)
```

All values in the updates list with zero/false/empty string are not added to the parameter list. We think the problem is this statement: https://github.com/envms/fluentpdo/blob/master/src/Queries/Base.php#L320

FluentPDO version is 1.1.2.

**Comment by ErwinSteffens** [comment]
We have a workaround for now which is to wrap the update list in an array like this:
```
$query = $queryHelper->fluent()
                     ->insertInto( 'my_table', $inserts )
                     ->onDuplicateKeyUpdate( [ $updates ] );
```

**Comment by ErwinSteffens** [comment]
Sorry, my fault, the workaround does not work either.

**Comment by cbornhoft** [comment]
Hey @ErwinSteffens, thanks for submitting this. Looks like you are indeed correct, that line will remove anything "falsey". I'll have a look sometime this week and see if I can release a patch.

**Comment by cbornhoft** [comment]
Fixed in 2.0
